requirements.txt

```diff
@@ -1,4 +1,6 @@
 langchain-core==0.1.35
 langchain-community==0.0.28
-google-cloud-firestore==2.15.0
+# This is a fork of the original package that adds support for vector search
```

Comment:
Looks like this with be version 2.16.0 when released https://github.com/googleapis/python-firestore/pull/876